src/App.js

```diff
@@ -8,6 +8,7 @@ import { Route, Routes } from "react-router-dom";
 
 function App() {
   const [reviews, setReviews] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
Do we need to hold a single isLoading state this high up in app? This would mean that the entire app would be set to loading each time the state was changed anywhere else across the page. It is usually best to hold this state as low as possible so that only the component that is actually loading has the messaging attached to it.

Similarly with you reviews state here, do we need to be holding this in App? Remember we always want to be holding state as low as possible.